index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Online Course capstone</title>
+    <meta
+      name="description"
+      content="Online Course for capstone project of html, css and javascript"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="./bootstrap/css/bootstrap.css"
+    />
+    <link rel="stylesheet" type="text/css" href="./Css/style.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header col-12">
```

Comment:
- [ ] There is a white space by the right side of your homepage in the desktop version. let's fix this overflow to make the page look professional and to the page's UI.

<img width="587" alt="Loom_90eNMF19lg" src="https://user-images.githubusercontent.com/81468675/152481307-0d52bfed-f36e-4576-b479-2bb5e9c4a627.png">